locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -665,42 +677,55 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
             setup.putIfNotNull(JcloudsLocationConfig.PRIVATE_KEY_DATA, userCredentials.getOptionalPrivateKey().orNull());
 
             // Wait for the VM to be reachable over SSH
-            if (waitForSshable) {
+            if (waitForSshable && !windows) {
                 waitForReachable(computeService, node, sshHostAndPortOverride, userCredentials, setup);
             } else {
                 LOG.debug("Skipping ssh check for {} ({}) due to config waitForSshable=false", node, setup.getDescription());
             }
             usableTimestamp = Duration.of(provisioningStopwatch);
 
+            JcloudsSshMachineLocation jcloudsSshMachineLocation = null;
```

Comment:
I wonder about just having the `machineLocation` variable, and casting it where required. Rather than having 3 variables where 1 will be null. Not sure what the best pattern is for that. The casting would force us to think about it more, to decide if we can generalise the method to take a machineLocation instead (and thus work with ssh and windows).